en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md

```diff
@@ -63,7 +63,7 @@ password = "$secret{keystore_password}"
 ```
 
 ### Passwords in LOG4j properties
-For example, consider the 'log4j.appender.LOGEVENT.password' in the log4j.properties file. You can refer the [encrypted password](#encrypt-passwords) from the log4j.properties file as shown below.
+For example, consider the 'log4j.appender.LOGEVENT.password' in the `log4j2.properties` file. You can refer the [encrypted password](#encrypt-passwords) from the `log4j2.properties` file as shown below.
```

Comment:
`log4j.appender.LOGEVENT.password` -> is this there in the new file?

Reply:
Will resolve this separately since we don't have a document with the updated content 
https://github.com/wso2/product-is/issues/11964

Reply:
in that case, shouldn't we hide it?